pandas/io/formats/csvs.py

```diff
@@ -188,6 +188,35 @@ def save(self):
                 for _fh in handles:
                     _fh.close()
 
+    def _index_label_encoder(self):
```

Comment:
there is `_get_column_name_list` in format.py, could this be moved into a base class/mixin and reused?

https://github.com/pandas-dev/pandas/blob/f57bd72310207b36e96604056f01cb77f0640b3b/pandas/io/formats/format.py#L840-L848

Reply:
Potentially.  Let's revisit once tests are passing again.

Reply:
future PR though.